util/find-doc-nits

```diff
@@ -344,8 +344,7 @@ getopts('lnshu');
 
 &help() if ( $opt_h );
 
-die "Need one of -l -n -s or -u flags.\n"
-    unless $opt_l or $opt_n or $opt_s or $opt_u;
+&help() unless $opt_l or $opt_n or $opt_s or $opt_u;
```

Comment:
Please remove this change.  If you really want it, make it a separate PR.  Personally, I find the change unhelpful.